dandi/dandiapi.py

```diff
@@ -850,10 +860,19 @@ def refresh(self) -> None:
         # Clear _version so it will be refetched the next time it is accessed
         self._version = None
 
-    def get_versions(self) -> Iterator[Version]:
-        """Returns an iterator of all available `Version`\\s for the Dandiset"""
+    def get_versions(self, order: Optional[str] = None) -> Iterator[Version]:
+        """
+        Returns an iterator of all available `Version`\\s for the Dandiset
+
+        Versions can be sorted by a given field by passing the name of that
+        field as the ``order`` parameter.  Currently, the only accepted field
+        name is ``"created"``.  Prepend a hyphen to the field name to reverse
+        the sort order.
+        """
         try:
-            for v in self.client.paginate(f"{self.api_path}versions/"):
+            for v in self.client.paginate(
+                f"{self.api_path}versions/", params={"order": order}
```

Comment:
ok then, lets proceed as is.
Just to make sure: this solution would still be compatible with currently deployed way to publish so it would be safe to release it into the wild right away, correct?

Reply:
Yes.